app/assets/javascripts/copo/copo.edit-checkin.es6

```diff
@@ -87,24 +144,29 @@ window.COPO.editCheckin = {
 
   updateCheckin(response) {
     // tries to find the checkin in gon and update it with the response
-    checkin = _.find(gon.checkins, _.matchesProperty('id',response.id));
-    checkin.lat = response.lat;
-    checkin.lng = response.lng;
-    checkin.edited = response.edited;
+    checkin = _.find(gon.checkins, _.matchesProperty('id', response.checkin.id));
+    checkin.lat = response.checkin.lat;
```

Comment:
@dmitrytrager I just meant having checkins be returned with all values (such as edited), not just the ones passed from serializer.